src/ekobox/apply.py

```diff
@@ -34,6 +33,7 @@ def apply_pdf(
         out_grid : dict
             output PDFs and their associated errors for the computed mu2grid
     """
+    qed = eko.theory_card.order[1] > 0
```

Comment:
here and in the following don't you need to cast to bool? 
```suggestion
    qed = bool(eko.theory_card.order[1] > 0)
```

Reply:
I can do it but I think it will not change anything: `order[1] > 0` is already a bool

Reply:
yes true, stupid me